parquet/src/main/java/org/apache/iceberg/parquet/Parquet.java

```diff
@@ -203,6 +210,16 @@ public WriteBuilder writerVersion(WriterVersion version) {
       return this;
     }
 
+    public WriteBuilder withFileEncryptionKey(ByteBuffer encryptionKey) {
+      this.fileEncryptionKey = encryptionKey;
+      return this;
+    }
+
+    public WriteBuilder withAadPrefix(ByteBuffer aadPrefix) {
```

Comment:
I think this should be `withAADPrefix`. We don't follow the Java convention of lower-casing commonly uppercase acronyms.